java/src/main/java/ai/rapids/cudf/Aggregation.java

```diff
@@ -300,12 +300,16 @@ public boolean equals(Object other) {
         }
     }
 
-    private static final class CollectSetAggregation extends Aggregation {
+    /**
+     * WARNING: https://github.com/rapidsai/cudf/issues/8405
```

Comment:
What?

Reply:
I agree that more of an explanation of what the issue is would be good.  I would like to keep the link because it can help people with traceability to know if it has been fixed or not.